matisse/src/main/res/values/strings.xml

```diff
@@ -32,6 +32,8 @@
     <string name="error_file_type">Unsupported file type</string>
     <string name="error_type_conflict">Can\'t select images and videos at the same time</string>
     <string name="error_no_video_activity">No App found supporting video preview</string>
+    <string name="error_over_original_size">Can\'t select the images larger than %1$d MB</string>
+    <string name="error_over_original_count">Can\'t select more than %1$d images or %2$d MB size</string>
```

Comment:
sorry ,  **error_over_original_count** 's  translate is not right . 
In fact ,it means  the selected image over the size , original checkbox will be canceled .please check your commit .